samples/msgext-northwind-inventory-python/.vscode/tasks.json

```diff
@@ -0,0 +1,111 @@
+// This file is automatically generated by Teams Toolkit.
+// The teamsfx tasks defined in this file require Teams Toolkit version >= 5.0.0.
+// See https://aka.ms/teamsfx-tasks for details on how to customize each task.
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Prepare Teams App Resources",
+            "dependsOn": [
+                "Validate prerequisites",
+                "Start local tunnel",
+                "Start Azurite emulator",
+                "Provision",
+                "Deploy"
+            ],
+            "dependsOrder": "sequence"
+        },
+        {
+            "type": "shell",
+            "label": "npm install",
+            "command": "npm install --no-audit"
+        },
```

Comment:
This task is not included in the `dependsOn` array in the previous task, therefore is not executed and so Azure package is not installed. The task should be executed before the `Start Azurite emulator` task.

Reply:
Fixed